benchmarks/README.md

```
@@ -0,0 +1,189 @@
+pnpm bench -- --runs 2 --libs tanstack,virtua --scenarios mount-fixed-10k
+
+# Watch the browser as it runs
+pnpm bench:headed
+```
+
+Results land in `benchmarks/results/<timestamp>.json` (raw, every run) and
+`benchmarks/results/LATEST.md` (median table from the last run).
+
+## How it works
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)